### DISCUSSION TITLE
Using a game controller as a fob.

### DISCUSSION THREAD

**Discussion by phil-barrett** [terjeio/ioSender#179] [Ideas]
While not my first choice, I have been playing with getting an XBox 360 controller working with ioSender.  I tried several applications - they all seemed to work but most required payment.  One that is free, [keysticks](https://keysticks.net/default.aspx), seems to work pretty well.  It is a little quirky but easy enough to get working (you have to think like a keyboard).  I will do a blog post for those who want to try it out.

Anyone else doing something similar?

**Comment by phil-barrett** [comment]
One problem I'm having is if you open the jogging panel and close it, kbd jog gets turned off and has to be re-enabled.  ctrl+shift turns it on, but only when the probe panel is open.  Is there another way to turn on keyboard jogging?

**Reply by calabr** [reply]
Probablr 3D window lost focuse - try to click central window.

**Reply by terjeio** [reply]
If an input field has focus keyboard jogging is disabled to allow normal editing, either click outside any input field or press the tab key until none has focus.

**Comment by calabr** [comment]
> Anyone else doing something similar?

my current profile with Antimicro
[usb_ioSender.gamecontroller.zip](https://github.com/terjeio/ioSender/files/7775295/usb_ioSender.gamecontroller.zip)
left hand jogging. Fast move with left trigger pressed, step with left shoulder
